Doc/library/json.rst

```diff
@@ -678,40 +678,43 @@ when serializing instances of "exotic" numerical types such as
 
 
 .. _json-commandline:
-.. program:: json.tool
+.. program:: json
 
-Command Line Interface
+Command-Line Interface
 ----------------------
 
-.. module:: json.tool
-    :synopsis: A command line to validate and pretty-print JSON.
-
 **Source code:** :source:`Lib/json/tool.py`
 
 --------------
 
-The :mod:`json.tool` module provides a simple command line interface to validate
-and pretty-print JSON objects.
+The :mod:`json` module can be invoked as a script,
+using the interpreter's ``-m`` switch,
+to validate and pretty-print JSON objects.
```

Comment:
Perhaps (more directly copy-and-pasteable)
```suggestion
The :mod:`json` module can be invoked as a script via ``python -m json``
to validate and pretty-print JSON objects.
```

Reply:
Thanks!